### PR TITLE
Core: cleanups

### PR DESCRIPTION
**Pull request by yuwata**
These commits are initially introduced in PR #6497. But they are independent of the PR. So, I just separate PR.

### CODE REVIEW

File: src/core/execute.c [hunk 1]
```diff
@@ -3524,23 +3524,23 @@ void exec_context_dump(ExecContext *c, FILE* f, const char *prefix) {
...
 
-                ioprio_class_to_string_alloc(IOPRIO_PRIO_CLASS(c->ioprio), &class_str);
-                fprintf(f,
-                        "%sIOSchedulingClass: %s\n"
-                        "%sIOPriority: %i\n",
-                        prefix, strna(class_str),
-                        prefix, (int) IOPRIO_PRIO_DATA(c->ioprio));
+                r = ioprio_class_to_string_alloc(IOPRIO_PRIO_CLASS(c->ioprio), &class_str);
+                if (r >= 0)
+                        fprintf(f, "%sIOSchedulingClass: %s\n", prefix, strna(class_str));
```
Comment (keszybz) [inline src/core/execute.c]: We don't need the `strna` now, right?

File: src/core/execute.c [hunk 2]
```diff
@@ -3524,23 +3524,23 @@ void exec_context_dump(ExecContext *c, FILE* f, const char *prefix) {
...
+
+                fprintf(f, "%sIOPriority: %i\n", prefix, (int) IOPRIO_PRIO_DATA(c->ioprio));
```
Comment (keszybz) [inline src/core/execute.c]: Hmm, while at it, maybe change the format to "%ul", and drop the cast?

File: src/core/execute.c [hunk 3]
```diff
@@ -3524,23 +3524,23 @@ void exec_context_dump(ExecContext *c, FILE* f, const char *prefix) {
...
         }
 
         if (c->cpu_sched_set) {
                 _cleanup_free_ char *policy_str = NULL;
 
-                sched_policy_to_string_alloc(c->cpu_sched_policy, &policy_str);
+                r = sched_policy_to_string_alloc(c->cpu_sched_policy, &policy_str);
+                if (r >= 0)
+                        fprintf(f, "%sCPUSchedulingPolicy: %s\n", prefix, strna(policy_str));
```
Comment (keszybz) [inline src/core/execute.c]: `strna` again.

File: src/core/execute.c [hunk 4]
```diff
@@ -3590,14 +3590,13 @@ void exec_context_dump(ExecContext *c, FILE* f, const char *prefix) {
 
                 _cleanup_free_ char *fac_str = NULL, *lvl_str = NULL;
 
-                log_facility_unshifted_to_string_alloc(c->syslog_priority >> 3, &fac_str);
-                log_level_to_string_alloc(LOG_PRI(c->syslog_priority), &lvl_str);
+                r = log_facility_unshifted_to_string_alloc(c->syslog_priority >> 3, &fac_str);
+                if (r >= 0)
+                        fprintf(f, "%sSyslogFacility: %s\n", prefix, strna(fac_str));
```
Comment (keszybz) [inline src/core/execute.c]: And here.

File: src/core/execute.c [hunk 5]
```diff
@@ -3590,14 +3590,13 @@ void exec_context_dump(ExecContext *c, FILE* f, const char *prefix) {
...
 
-                fprintf(f,
-                        "%sSyslogFacility: %s\n"
-                        "%sSyslogLevel: %s\n",
-                        prefix, strna(fac_str),
-                        prefix, strna(lvl_str));
+                r = log_level_to_string_alloc(LOG_PRI(c->syslog_priority), &lvl_str);
+                if (r >= 0)
+                        fprintf(f, "%sSyslogLevel: %s\n", prefix, strna(lvl_str));
```
Comment (keszybz) [inline src/core/execute.c]: And here.

**Comment by yuwata** [comment]
@keszybz Thank you for your comments. Updated.

**Comment by keszybz** [comment]
```
../src/core/execute.c:3531:17: error: format ‘%u’ expects argument of type ‘unsigned int’, but argument 4 has type ‘long unsigned int’ [-Werror=format=]
                 fprintf(f, "%sIOPriority: %ul\n", prefix, IOPRIO_PRIO_DATA(c->ioprio));
                 ^
```
→ `%lu`

**Comment by yuwata** [comment]
Sorry... Updated.

**Comment by keszybz** [comment]
I suggested it ;) Anyway, looks good now.

**Pull request merged**